ovs/datapath.go

```diff
@@ -0,0 +1,338 @@
+// Copyright 2021 DigitalOcean.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ovs
+
+import (
+	"errors"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var (
+	errMissingMandatoryDataPathName = errors.New("datapath name argument is mandatory")
+	errUninitializedClient          = errors.New("client unitialized")
+	errMissingMandatoryZone         = errors.New("at least 1 zone is mandatory")
+	errWrongArgumentNumber          = errors.New("missing or too many arguments to setup ct limits")
+	errWrongDefaultArgument         = errors.New("wrong argument while setting default ct limits")
+	errWrongZoneArgument            = errors.New("wrong argument while setting zone ct limits")
+)
+
+// Zone defines the type used to store a zone as it is returned
+// by ovs-dpctl ct-*-limits commands
+type Zone map[string]uint64
```

Comment:
nit: here it could be interesting to name this map "CTLimit" since it's holding "default" and "zone" limits